user_sync/app.py

```diff
@@ -353,6 +359,17 @@ def begin_work(config_loader):
     directory_groups = config_loader.get_directory_groups()
     rule_config = config_loader.get_rule_options()
 
+    ssl_cert_verify = rule_config.get('ssl_cert_verify', True)
```

Comment:
While we normally prefer the `get()` dict method, I don't think it makes sense to use it here. Since the `ssl_cert_verify` key originates from the `invocation_defaults` dict, we are safe to assume the key is present here. If the key doesn't exist for some reason, we want to get the error instead of silently providing a default value.